docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,226 @@
+A list of all the available aliases can be obtained by the `testcafe --list-browsers` command (see [below](#b)).
+
+The following example demonstrates how to run a test in two browsers: Internet Explorer and Google Chrome.
+
+```sh
+testcafe ie,chrome c:\Tests\SampleFixture.test.js
+```
+
+### All Browsers
+
+To run tests against all installed browsers, use the *all* alias.
+
+```sh
+testcafe all c:\Tests\SampleFixture.test.js
+```
+
+### <a class="anchor" name="remotes"></a>Remote Browsers
+
+To run tests on a remote browser, specify *remote* as a browser alias. 
+
+```sh
+testcafe remote c:\Tests\SampleFixture.test.js
+```
+
+TestCafe will provide a URL that you need to open in the desired browser on your remote device. 
+After that, your remote browser will be connected to TestCafe and tests will start.
+
+If you want to connect multiple browsers,
+prefix alias with the number of browsers you want to connect. For example, if you need to use three remote browsers, specify *3remote*.
+
+```sh
+testcafe 3remote c:\Tests\SampleFixture.test.js
+```
+ 
+TestCafe will provide an individual URL for each browser.
+Note that tests will start only after all the browsers are connected.
+
+## File Path/GLOB Pattern
+
+Specify one or more file paths to run tests from. If you specify several files, separate them by a space.
+
+```sh
+testcafe ie c:\Tests\Fixture1.test.js c:\Tests\Fixture2.test.js
+```
+
+You can use globbing patterns to specify a set of files. 
+For example, the following command runs tests from files that matches *c:\Tests\&#42;Page&#42;.js*, namely, *c:\Tests\ExamplePage.test.js* and *c:\Tests\MainPage.test.js*.
+
+```sh
+testcafe ie c:\Tests\*Page*js
+```
+
+You can also specify only a directory to run tests from.
+
+```sh
+testcafe ie c:\Tests\
+```
+
+If you do not specify any file or directory, TestCafe will run tests from files located in the */TestCafe/Tests* directory.
+
+## Options
+
+### -h  
+### --help  
+Display usage information for commands.
+
+```sh
+testcafe --help
+```
+
+### <a class="anchor" name="b"></a>-b  
+### --list-browsers 
+Lists all browsers available on your system.
+
+```sh
+testcafe --list-browsers
+```
+
+### -r \<name\> 
+### --reporter \<name\>  
+Specify a built-in or custom reporter that will be used to generate test reports. 
+The available built-in reporters are *json*, *list*, *minimal*, *spec*, *xunit*. By default, the *spec* reporter is used.
+
+The following command tells TestCafe to run the tests in all available browsers and generate a report in the xunit format.
+
+```sh
+testcafe all "c:\Tests\SampleFixture.test.js" -r xunit
+```
+
+A generated test report will be displayed in the command prompt window. 
+
+If you need to save a report to an external file, you can redirect the command output stream to a file  
+by using the > redirection operator and specifying a report file path.
+
+```sh
+testcafe all c:\Tests\SampleFixture.test.js > c:\Tests\test-results.txt
+```
```

Comment:
> c:\Tests\test-results.txt

Here and after: use relative paths, since ` c:\Tests\test-results.txt` is windows specific. Prefere lower-cased file and dir names:
```sh
testcafe all /tests/sample-fixture.js > /tests/test-results.txt
```

Don't use `.test.js` as extension for test files, it should be `.js`